groups/wg-productivity/groups.yaml

```diff
@@ -56,18 +54,17 @@ groups:
       ReconcileMembers: "true"
       WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
     members:
-      - chizhg@google.com
       - kmahapatra@vmware.com
+      - cy@borg.dev # Mahamed Ali
 
-  - email-id: k8s-infra-rbac-perf-tests@knative.dev
-    name: k8s-infra-rbac-perf-tests
-    description: |-
-      Grants access to the shared community cluster perf-tests namespace
-    settings:
-      ReconcileMembers: "true"
-      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
-    members:
-      - nziada@vmware.com
+  # - email-id: k8s-infra-rbac-perf-tests@knative.dev
+  #   name: k8s-infra-rbac-perf-tests
+  #   description: |-
+  #     Grants access to the shared community cluster perf-tests namespace
+  #   settings:
+  #     ReconcileMembers: "true"
+  #     WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW" # required
+  #   members:
```

Comment:
wasn't sure if this group should just be deleted as there are no members...

Reply:
@upodroid set this up for Nader and others to use the community clusters. I don't know if this is still needed though, is it @upodroid ?

If not then yes lets remove it.

Reply:
I granted nader out of band access. I need to add him to the group and then grant the permissions to the group.

Reply:
sounds like leaving it commented out until it is used is the right approach. In other words, looks good the way you have it.